js/glowingbear.js

```diff
@@ -727,6 +727,26 @@ weechat.controller('WeechatCtrl', ['$rootScope', '$scope', '$store', '$timeout',
         settings.nonicklist = !settings.nonicklist;
     };
 
+    $rootScope.switchToAdjacentBuffer = function(direction) {
+        // direction is +1 for next buffer, -1 for previous buffer
+        var sortedBuffers = _.sortBy($scope.getBuffers(), 'fullName');
```

Comment:
this should probably mimic what the angular stuff does in the html and use `$rootScope.predicate` for sorting instead of hardcoded `'fullName'`